web/shapeworks/src/store.ts

```diff
@@ -55,3 +55,10 @@ export const loadGroomedShapeForObject = async (type: string, id: number) => {
     }
     groomedShapesForOriginalDataObjects.value[type][id] = groomed
 }
+
+export function submitForm(action:string, payload: Record<string, any>){
+    if (Object.keys(payload).every((key) => key.includes("section"))) {
+        payload = Object.assign({}, ...Object.values(payload))
+    }
+    console.log(action, payload)
```

Comment:
what does this function actually do?

Reply:
So far, nothing. This is where we would hook it up to corresponding API calls.